src/mol-model-formats/structure/property/bonds/index-pair.ts

```diff
@@ -45,27 +47,42 @@ export namespace IndexPairBonds {
     export type Data = {
         pairs: {
             indexA: Column<number>,
-            indexB: Column<number>
+            indexB: Column<number>,
+            id?: Column<number>,
```

Comment:
Should this be called ``key`` instead? I think it would better reflect what it is for.

Reply:
ok, changed it to key